tests/test_images.sh

```diff
@@ -251,8 +251,9 @@ setup()
 	[ ! -d "${tests_repo_dir}" ] && git clone "https://${tests_repo}" "${tests_repo_dir}"
 
 	if [ -z "${KATA_DEV_MODE:-}" ]; then
-		chronic $mgr remove-docker
-		chronic $mgr install-docker-system
+		"${tests_repo_dir}/.ci/setup.sh"
+		mkdir -p /etc/kata-containers/
+		sudo cp -a /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers/configuration.toml
```

Comment:
Why is needed to copy /etc/kata-containers/ ?

Reply:
I think this is because the script makes use of the `kata-manager.sh` script elsewhere. And that script assumes you have a local `/etc/kata-containers/configuration.toml` file that it can make changes to.

Reply:
@jodh-intel you're right